install.ps1

```diff
@@ -196,12 +196,12 @@ function Install-Mondoo {
   }
 
   # show notification to change execution policy:
-  if((Get-ExecutionPolicy) -gt 'RemoteSigned' -or (Get-ExecutionPolicy) -eq 'ByPass') {
-    fail "
-  PowerShell requires an execution policy of 'RemoteSigned'. Please change the policy by running:
-  Set-ExecutionPolicy RemoteSigned -scope CurrentUser
-  "
-  }
+  # if((Get-ExecutionPolicy) -gt 'RemoteSigned' -or (Get-ExecutionPolicy) -eq 'ByPass') {
```

Comment:
Let's completely remove that part since we do not need it anymore once signing is enabled.